docker-compose/docker-compose-injistack/docker-compose.yaml

```diff
@@ -14,7 +14,7 @@ services:
       - "5433:5432"
 
   certify:
-    image: mosipqa/inji-certify-with-plugins:develop
+    image: mosipqa/inji-certify-with-plugins:0.12.x
```

Comment:
I believe this and other changes related to version change in pom is not required.